src/ceph-volume/ceph_volume/devices/lvm/batch.py

```diff
@@ -143,7 +143,7 @@ def __init__(self, argv):
         parser = argparse.ArgumentParser(
             prog='ceph-volume lvm batch',
             formatter_class=argparse.RawDescriptionHelpFormatter,
-            description=self.print_help(),
+            description="Utility for deploying osd in batch.",
```

Comment:
Please keep using the help text in `self._help`. Just drop the format and remove the placeholder at https://github.com/ceph/ceph/blob/b86b073b919ee22e80edec72d1ef6daed65dfa09/src/ceph-volume/ceph_volume/devices/lvm/batch.py#L131 plus the two surrounding lines.

Reply:
done in the updated push.